packages/web-components/src/text-area/text-area.spec.md

```diff
@@ -0,0 +1,69 @@
+# Text area
+
+## Component Description
+
+A text area is a type of input field that allows users to enter multiple lines of text. It is commonly used in web forms and text editors to provide a space for users to input large amounts of text, such as comments, messages, or other types of content.
+
+## Design Spec
+
+[Link to Design Spec in Figma](https://www.figma.com/file/IBCBJxEbPKS7CvLHG55Wn9/Slider?node-id=513%3A510&t=hCOhOiQJVCVNVVJP-0)
+
+## Engineering Spec
+
+### Inputs
+
+- @attr autocomplete: "on" "off" | "off"
+- @attr autofocus: boolean
+- @attr cols: number
+- @attr value: string
+- @attr disabled: boolean | false
+- @attr name: string
+- @attr placeholder: string
+- @attr resize: "none" "both" "horizontal" "vertical" | "none"
+- @attr rows: number
+- @attr minlength: number
+- @attr maxlength: number
+
```

Comment:
We'll want to add 'appearance' here to mimic the [Fluent textarea](https://master--628d031b55e942004ac95df1.chromatic.com/?path=/docs/components-textarea--default). The values are: `"outline" "filled-darker" "filled-lighter" "filled-darker-shadow" "filled-lighter-shadow"`

Similarly we should add `size`: `"small" "medium" "large"`